### DISCUSSION TITLE
Only 200 Stars is a crime for this library

### DISCUSSION THREAD

**Discussion by armanckeser** [vincjo/datatables#65] [General]
I just wanted to say that I really appreciate this library. It's approach to data is incredibly clean, allows for infinite extensibility and customization, and it just works well. Kudos to you for maintaining and developing it. A truly underrated library.

**Comment by vincjo** [comment]
Thank you so much!

**Comment by yayza** [comment]
Just wanted to drop in and say the new site redesign looks great 👍
